uzdevums_3.py

```diff
@@ -0,0 +1,4 @@
+# Aprēķināt riņķa līnijas garumu, ja tiek ievadīts tās rādiuss.
+from math import pi
+r = float(input ("Ievadi rinka linijas radiusu, lai apreikinatu garumu : "))
+print ("Rinka linijas garums ar " + str(r) + " ir: " + str(pi * r**2))
```

Comment:
Bonuss par `math.pi` izmantošanu.
Bet līnijas garumu aprēķina `2 * pi * R`, nevis `pi * R * R`.
Un starp mainīgajiem un darbībām liekam atstarpi:
```suggestion
print("Rinka linijas garums ar " + str(r) + " ir: " + str(pi * r * 2))
```